code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -89,6 +91,47 @@
 	..()
 	return TRUE
 
+/datum/reagent/medicine/C2/probital
+	name = "Probital"
+	description = "Originally developed as a prototype-gym supliment for those looking for quick workout turnover, this oral medication quickly repairs broken muscle tissue but causes lactic acid buildup, tiring the patient. Overdosing can cause extreme drowsiness. An Influx of nutrients promotes the muscle repair even further."
+	reagent_state = SOLID
+	color = "#FFFF6B"
+	overdose_threshold = 20
+
+/datum/reagent/medicine/C2/probital/on_mob_life(mob/living/carbon/M)
+	M.adjustBruteLoss(-2.25*REM, FALSE)
+	var/ooo_youaregettingsleepy = 3.5
```

Comment:
I don't like this variable name

Reply:
oooooOOoOOoo you want to resolve this review